app/modules/UtilityModule.py

```diff
@@ -0,0 +1,15 @@
+import json
+import subprocess
+from pymongo import MongoClient
+
+class Rescan(object):
+
+	def rescan(self):
```

Comment:
I don't see any of your new code that calls `rescan` Did I miss it?

Reply:
Removed